container/kvm/wrappedcmds_internal_test.go

```diff
@@ -123,6 +125,39 @@ func (libvirtInternalSuite) TestWriteDomainXMLMissingBothDisk(c *gc.C) {
 	c.Assert(got, gc.Matches, "")
 }
 
+func (libvirtInternalSuite) TestCreateNVRAMOnARM64(c *gc.C) {
+	d, err := ioutil.TempDir("", "juju-libvirtInternalSuite-")
```

Comment:
Primarily ignorance. However, now that you mention it, I recall seeing it. But I guess it didn't register properly... changed them all in here.